Fix: styles marked dates

hellpirat opened the pull request:
Fix styles, update screenshot and version 

![image](https://user-images.githubusercontent.com/6241354/50537547-f1f3af80-0b71-11e9-806d-2ca3294f8b2e.png)

peacechen commented:
Thanks again @hellpirat 

I did a minor refactor to `renderDots()` so that the `markedDates` check is inside the method instead of inline with the JSX.  That makes it cleaner and easier read in the render method.

It dawned on me that React.Fragment was introduced in 16.2.0.  There are numerous users of this library who may be using older versions of React, and this library shouldn't break their usage.  I've changed it to a traditional `View` wrapper.  Please validate on your end that everything works as it should

hellpirat commented:
@peacechen 

There are a error:  Invariant Violation: Invariant Violation: Invariant Violation: Objects are not valid as a React child (found: object with keys {key, color, selectedDotColor}). If you meant to render a collection of children, use an array instead.

Because of map and filter used.

I fixed it and also fixed styles because now we use <View> instead of <React.Fragment> :)

There is "react": "^16.2.0" in devDependencies.  🤔

peacechen commented:
Thanks for testing the changes. 

The peerDependency version `*` is what consumers of this library base the dependencies on.  The devDependency is for running tests locally for this library.  I'm not sure when that was updated to 16.2.0, but it was probably to verify that this continues to work with the latest version of React.
